force-app/main/default/classes/SOQL.cls

```diff
@@ -201,7 +224,8 @@ public with sharing class SOQL {
      * @return A list of Cases
      */
     public static List<Case> getCasesWithStatus() {
-        return null; // Replace null with the variable you used to store the result
+        List<Case> getCases = [SELECT Status, CreatedDate FROM Case WHERE Status = 'Open' ORDER BY CreatedDate LIMIT 10 OFFSET 5];
```

Comment:
Try to filter by what's not closed, since there are some other statuses on case that might be considered Open